pom.xml

```diff
@@ -428,6 +430,24 @@
 				<version>${aws.sqs.version}</version>
 			</dependency>
 
+			<dependency>
+				<groupId>com.atlassian.jira</groupId>
+				<artifactId>jira-rest-java-client-core</artifactId>
+				<version>${jira.client.version}</version>
+			</dependency>
+
+			<dependency>
+				<groupId>com.atlassian.jira</groupId>
+				<artifactId>jira-rest-java-client-api</artifactId>
+				<version>${jira.client.version}</version>
+			</dependency>
+
+			<dependency>
+				<groupId>io.atlassian.fugue</groupId>
+				<artifactId>fugue</artifactId>
+				<version>${jira.fugue.version}</version>
+			</dependency>
+
```

Comment:
this stuff was already added in https://github.com/syndesisio/syndesis-qe/pull/1073